package.json

```diff
@@ -65,7 +65,7 @@
     "babel-loader": "^6.2.10",
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-external-helpers": "^6.22.0",
-    "babel-plugin-flow-react-proptypes": "^0.18.2",
+    "babel-plugin-flow-react-proptypes": "^1.2.0",
```

Comment:
It should use `^2.1.3` now (**@see** https://github.com/brigand/babel-plugin-flow-react-proptypes/pull/81)